tests/common.go

```diff
@@ -1519,6 +1558,12 @@ func ValidateApplicationsPurePxctl(contexts []*scheduler.Context) {
 func ValidateApplicationsPureSDK(contexts []*scheduler.Context) {
 	Step("validate applications", func() {
 		for _, ctx := range contexts {
+			if ctx.App.Key == "wordpress" || ctx.App.Key == "nginx-fa-davol" {
```

Comment:
This will not be necessary once we remove the `SkipMountOptionValidation` variable, from my earlier comment.

Reply:
Done